docs/src/content/docs/fr/reference/configuration.md

```diff
@@ -123,6 +123,34 @@ starlight({
 Les groupes de barres latérales générées automatiquement sont triés par nom de fichier et par ordre alphabétique.
 Par exemple, une page générée à partir de `astro.md` apparaîtrait au-dessus de la page de `starlight.md`.
 
+#### Groupes rétractables
+
+Les groupes de liens sont développés par défaut. Vous pouvez modifier ce comportement en définissant la propriété `collapsed` d'un groupe sur `true`.
```

Comment:
I wonder if “développé” is right for an “expanded” group or whether something like “étendu” would be more appropriate?

Reply:
Aussi on peut utiliser expansé ou étendu, quelle sera-t-il la meilleure façon?

Reply:
Personally, I have only encountered "développé" / "réduit" in this context. "étendu" feels a bit weird to me, it's a bit difficult for me to explain but this sounds more like relative to the depth rather than a toggle between visible and hidden. And "expansé" is wrong in this context.

Some examples I found in the [Adobe documentation](https://helpx.adobe.com/fr/incopy/using/workspace-basics.html):

> Pour réduire ou développer toutes les icônes de panneau d’une colonne

> Pour développer une seule icône de panneau

> Pour réduire à la taille d’une icône un panneau qui a été développé

Reply:
So, should I rollback @delucis?

Reply:
Yes, let’s stick with “développé” then!